### DISCUSSION TITLE
Example of updating Modal container width

### DISCUSSION THREAD

**Discussion by captDaylight** [chakra-ui/chakra-ui#8211] [General]
Hi I'm trying to set the max width on the modal with this theming :

```
const customTheme = extendTheme({
  components: {
    Modal: {
      baseStyle: {
        dialog: {
          maxWidth: '30rem'
        }
      },
      sizes: {
        xl: {
          Content: {
            maxWidth: '30rem'
          }
        }
      }
    }
  },
});
```

But it's still showing 36rem wide:
<img width="329" alt="Screen Shot 2024-01-12 at 5 03 16 PM" src="https://github.com/chakra-ui/chakra-ui/assets/700617/34ffeac5-5bf8-4504-890c-75d5b2c23683">

The only thing that works is :
```
<ModalContent maxW="30rem">
```

But I don't want to write this locally everywhere.

**Comment by segunadebayo** [comment]
Modal is `Dialog` now. Set width on `Dialog.Content`.
